corehq/apps/app_manager/models.py

```diff
@@ -2514,6 +2514,16 @@ class ApplicationBase(VersionedDoc, SnapshotMixin,
     build_langs = StringListProperty()
     secure_submissions = BooleanProperty(default=False)
 
+    # metadata for data platform
+    amplifies_workers = StringProperty(
```

Comment:
should be called `is_wammy`. :)

![image](https://cloud.githubusercontent.com/assets/66555/5987942/736621bc-a94f-11e4-8725-3a3a9b3410c3.png)

@hartfordr